src/pages/ItemList/Components/Side.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import './Side.scss';
+
+const Side = ({ renderCategory }) => {
+  const [whichSelected, setwhichSelected] = useState('teashop');
+
+  return (
+    <div className="side">
+      <p className="listTitle">제품</p>
+      <ul className="firstMenu">
+        <li
+          data-category="first-category"
+          id="1"
+          onClick={e => {
+            setwhichSelected('teashop');
+            renderCategory(e);
+          }}
+          className={
+            whichSelected === 'teashop'
+              ? 'activatedTabTeaShop'
+              : 'unactivatedTabTeaShop'
+          }
+        >
+          티 제품
+          <div className="secondMenu1">
+            <ul>
+              <li
+                data-category="first-category"
+                id="1"
+                onClick={e => renderCategory(e)}
```

Comment:
이벤트 객체를 전달하기 위해서라면 그냥 해당 함수만 적어주셔도 전달됩니다.
[이벤트 처리하기](https://ko.reactjs.org/docs/handling-events.html)
```suggestion
                onClick={renderCategory}
```